components/global/mixer-adapter/cmd/mixer-adapter/main.go

```diff
@@ -58,46 +100,123 @@ func main() {
 		}
 	}()
 
-	if len(os.Args) > 1 {
-		port, err = strconv.Atoi(os.Args[1])
-		if err != nil {
-			logger.Errorf("Could not convert the port number from string to int : %s", err.Error())
-		}
+	data, err := ioutil.ReadFile(os.Getenv(configFilePathEnv))
+	if err != nil {
+		logger.Fatal("Could not read the config file")
+	}
+	config := &Config{}
+	err = json.Unmarshal(data, &config)
+	if err != nil {
+		logger.Fatal("Could not unmarshal the data in config file")
 	}
 
-	/* Mutual TLS feature to secure connection between workloads
-	   This is optional. */
-	adapterCertificate := os.Getenv(grpcAdapterCertificatePath) // adapter.crt //change the name
-	adapterprivateKey := os.Getenv(grpcAdapterPrivateKeyPath)   // adapter.key
-	caCertificate := os.Getenv(caCertificatePath)               // ca.pem
-	spServerUrl := os.Getenv(spServerUrlPath)
-
-	logger.Infof("Sp server url : %s", spServerUrl)
+	// Mutual TLS feature to secure connection between workloads. This is optional.
+	adapterCertificate := config.Mixer.TLS.Certificate // adapter.crt
+	adapterPrivateKey := config.Mixer.TLS.PrivateKey   // adapter.key
+	caCertificate := config.Mixer.TLS.CaCertificate    // ca.pem
+	// Initialize the variables from the config map
+	spServerUrl := config.SpEndpoint.URL
+	filePath := config.Store.FileStorage.Path
```

Comment:
File path is specific to file based persistence method. Shall we move this logic to the relevant persistence implementation?